### DISCUSSION TITLE
Rolling update takes down all nodes

### DISCUSSION THREAD

**Discussion by ShuaiShao93** [rabbitmq/cluster-operator#1726] [Q&A] [answered]
## Describe the bug
When rolling update happens and one node is down, the other node is stopped and fails the readiness probe:
<img width="892" alt="image" src="https://github.com/user-attachments/assets/d9591d4d-0149-4d47-b6ee-5272350addf8">

## To Reproduce
1. Use operator to run a cluster of 2 nodes
2. Kill the pod of one of the nodes
3. Check the other node
4. See it fails the readiness probe

**Expected behavior**
The other node shouldn't be impacted when one node is down

## Screenshots
<img width="1105" alt="image" src="https://github.com/user-attachments/assets/1bbc6b70-f29d-4fc1-915e-2842debd10d5">

## Version and environment information
- RabbitMQ Cluster Operator: v2.10.0

**Comment by lukebakken** [comment, marked as answer]
Use 3 nodes, not 2.

**Reply by ShuaiShao93** [reply]
Nice! Verified this works!

Is this documented anywhere?

**Reply by lukebakken** [reply]
https://www.google.com/search?q=site%3Arabbitmq.com+cluster+size
